tests/copp/scripts/update_copp_config.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python
+"""
+    Script to modify a COPP configuration to follow a specified rate limit.
+
+    This is used by the COPP tests to reduce the rate limit below that of the
+    PTF host's sending rate.
+
+    Example::
+
+        $ python update_copp_config.py <rate limit in pps> <input file> <output file>
+        $ python update_copp_config.py 600 /tmp/copp_config.json /tmp/new_copp_config.json
+"""
+import json
+import sys
+
+def generate_limited_pps_config(pps_limit, input_config_file, output_config_file):
+    """
+        Modifies a COPP config to use the specified rate limit.
```

Comment:
noticed disparity in how comments are indented here when compared to other methods such as configure_syncd

Reply:
Oops, you're right! I'll make it consistent.